.github/workflows/configured_test.yml

```diff
@@ -0,0 +1,66 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+name: Run specific PDO tests
+on:
+  workflow_dispatch:
+    inputs:
+      interpreter:
+        description: 'Interpreter'
+        required: true
+        default: 'wawaka'
+        type: choice
+        options:
+        - wawaka
+        - wawaka-opt
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+        type: choice
+        options:
+          - debug
+          - info
+          - warning
+      memoryConfiguration:
+        description: 'Interpreter memory configuration'
+        required: false
+        default: MEDIUM
+        type: choice
+        options:
+          - SMALL
+          - MEDIUM
+          - LARGE
+jobs:
+    pdo_specific_tests:
+        name: Run specific PDO tests
+        runs-on: ubuntu-22.04
```

Comment:
This could be another parameter (input)?  Although admittedly as everything is done in docker, if we would parameterize that it probably would be better to parameterize docker's `UBUNTU_VERSION` build argument?

Reply:
good point. if i can actually use the variables from the dispatch i will add this.  it will probably have to be fixed in a later PR though since i have no way to test it right now.